packages/plugin-google-analytics/src/index.js

```diff
@@ -1,40 +1,85 @@
-module.exports = (options = {}) => {
-  const { analyticsId, anonymous } = options;
+const fs = require('fs');
+const TransformInterface = require('@greenwood/cli/src/plugins/transforms/transform.interface');
 
-  const validId = analyticsId && typeof analyticsId === 'string';
-  const trackAnon = typeof anonymous === 'boolean' ? anonymous : true;
+let options = '';
 
-  if (!validId) {
-    throw new Error(`Error: analyticsId should be of type string.  get "${typeof analyticsId}" instead.`);
+class GoogleAnalyticsPlugin extends TransformInterface {
+  
+  constructor(req, compilation) {
+    super(req, compilation, {
+      extensions: ['.html', '.md'], 
+      contentType: 'text/html'
+    });
+    
+    const { analyticsId, anonymous } = options;
+    this.analyticsId = analyticsId;
+
+    const validId = analyticsId && typeof analyticsId === 'string';
+    this.trackAnon = typeof anonymous === 'boolean' ? anonymous : true;
+
+    if (!validId) {
+      throw new Error(`Error: analyticsId should be of type string.  get "${typeof analyticsId}" instead.`);
+    }
+  }
+
+  shouldTransform() {
+    const { request, workspace } = this;
```

Comment:
should these two plugins only need to check for _.html_ or does it have to also do it for _.md_ too?  Could these perhaps reuse the business logic of the our default transforms maybe?

Also, we should [avoid](https://github.com/ProjectEvergreen/greenwood/pull/462) doing this kind of logic, re: the `replace('.html')`
```
`${workspace}/pages${url.replace('.html', '')}`;
```